gcloud-java-examples/README.md

```diff
@@ -69,6 +69,23 @@ To run examples from your command line:
   $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DatastoreExample" -Dexec.args="my_name delete"
   ```
 
+  Here's an example run of `DnsExample`.
+  
+  Note that you have to enable the Google Cloud DNS API on the [Google Developers Console][developers-console] before running the following commands.
+  Note that the example creates and deletes dns records of type A only. Operations with other record types are not implemented in the example.
+  ```
+  $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DnsExample" -Dexec.args="create some-sample-zone elaborateexample.com. description"
+  $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DnsExample" -Dexec.args="get some-sample-zone"
```

Comment:
Since get and list (just below) will show the user approximately the same thing, I suggest including only one of them here.